package.json

```diff
@@ -50,5 +50,9 @@
     "tslint": "^6.1.0",
     "typescript": "^4.7.2"
   },
-  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e",
+  "resolutions": {
+    "@polkadot/util": "13.5.5",
+    "@polkadot/util-crypto": "13.5.5"
+  }
```

Comment:
Note: `resolutions` is used by Yarn to enforce a specific dependency version is used. The error this PR fixes was caused by incompatible `@polkadot/` packages. Forcing the version to resolve solves the bug.

Reply:
Exactly. Some of the dependencies were not updated, ex. w3f/test-utils still uses [pjs 12](https://github.com/w3f/test-utils-ts/blob/master/package.json#L22).